### PR TITLE
Remove dashboard jbuilder views.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: dashboard/app/views/callouts/show.json.jbuilder [hunk 1]
```diff
@@ -1 +0,0 @@
-json.extract! @callout, :element_id, :text, :created_at, :updated_at
```
Comment (ashercodeorg) [inline dashboard/app/views/callouts/show.json.jbuilder:1]: Note that `callout.text` does not exist, so hitting (e.g.) studio.code.org/callouts/1.json gives a 500.

https://app.honeybadger.io/projects/3240/faults/32645619

**Comment by ashercodeorg** [comment]
The CircleCI failures seem legitimate... Looking...

**Comment by ashercodeorg** [comment]
Believe I've fixed the tests. Is it bad that I'm now returning `head :create` rather than rendering? Granted, I'm not sure what was ever rendered...

**Comment by ashercodeorg** [comment]
Ping... PTAL.

File: dashboard/app/controllers/gallery_activities_controller.rb [hunk 2]
```diff
@@ -39,7 +39,7 @@ def create
       authorize! :save_to_gallery, @gallery_activity.user_level
 
       if @gallery_activity.save
-        render action: 'show', status: :created, location: @gallery_activity
+        return head :created
```
Comment (joshlory) [inline dashboard/app/controllers/gallery_activities_controller.rb:42]: Where does this route get called?  Is it an AJAX request?  From where?

Comment (ashercodeorg) [reply dashboard/app/controllers/gallery_activities_controller.rb:42]: My impression is that the route gets set [here](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/application_controller.rb#L206-L211) and called from [here](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/feedback.js#L358-L360).

Comment (joshlory) [reply dashboard/app/controllers/gallery_activities_controller.rb:42]: Aha, it doesn't even look at the returned data (or display an error if the POST fails).

**Comment by ashercodeorg** [comment]
Force-merged after rebasing with staging.

**Pull request merged**